File: LE6/d3-1.cpp
#include <iostream>
using namespace std;

int main() {
    string senha;

    cout << "Digite uma senha: ";
    cin >> senha;

    while (senha.length() < 8) {
        cout << "Senha inválida! Deve ter pelo menos 8 caracteres." << endl;
        cout << "Digite novamente: ";
        cin >> senha;
    }

    cout << "Senha válida!" << endl;

    return 0;
}

File: RP6/d3-1.java
import java.util.Scanner;

public class Ex3_V1 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String senha;
        System.out.print("Digite uma senha: ");
        senha = sc.nextLine();

        while (senha.length() < 8) {
            System.out.println("Senha inválida! Deve ter pelo menos 8 caracteres.");
            System.out.print("Digite novamente: ");
            senha = sc.nextLine();
        }

        System.out.println("Senha válida!");
    }
}